h2d/Flow.hx

```diff
@@ -54,6 +57,15 @@ class FlowProperties {
 
 }
 
+/**
+ * Container object which behaves like flexbox container in CSS.
+ * Could be usefull when you want to place objects without worrying about setting their coordinates manually 
+ * (for example, in for automatic layout of UI elements).
+ * This object will take care about it, so none of the added object will overlap others. Each object will be placed after another,
+ * and when `width` or `height` (depends on `isVertical` property value) becomes bigger than `maxWidth` or `maxHeight` 
+ * then new line or new column will be started to place next objects.
+ * You'll just need to call `rewlow()` method and all objects will be positioned for you.

```

Comment:
`rewlow`.

(Also try to avoid "you" in documentation.)

Reply:
ok, will try to rework it, but my English isn't good at all